server/plugin/api.go

```diff
@@ -359,9 +359,105 @@ func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.
 		return
 	}
 
-	attachment := &model.SlackAttachment{
-		Text: body.DetailedMessage.Markdown,
+	var attachment *model.SlackAttachment
+	switch body.EventType {
+	case constants.WorkItemCreated, constants.WorkItemUpdated, constants.WorkItemDeleted, constants.WorkItemCommented:
+		attachment = &model.SlackAttachment{
+			Text: body.DetailedMessage.Markdown,
+		}
+	case constants.PullRequestCreated, constants.PullRequestUpdated, constants.PullRequestMerged:
+		reviewers := "None" // When no reviewers are added to the pull request
+		for i := 0; i < len(body.Resource.Reviewers); i++ {
+			if i == 0 {
+				reviewers = ""
+			}
+
+			if i != len(body.Resource.Reviewers)-1 {
+				reviewers += fmt.Sprintf("%s, ", body.Resource.Reviewers[i].DisplayName)
+			} else {
+				reviewers += body.Resource.Reviewers[i].DisplayName
+			}
+		}
+
+		attachment = &model.SlackAttachment{
+			Pretext: body.Message.Markdown,
+			Title:   fmt.Sprintf("%d: %s", body.Resource.PullRequestID, body.Resource.Title),
+			Fields: []*model.SlackAttachmentField{
+				{
+					Title: "Target Branch",
+					Value: strings.Split(body.Resource.TargetRefName, "/")[2],
```

Comment:
are we always sure we get length of `body.Resource.TargetRefName` greater than or equal to 3?

Reply:
yes